### DISCUSSION TITLE
How to enable all debug which is available?

### DISCUSSION THREAD

**Discussion by macktab** [tiangolo/uvicorn-gunicorn-docker#226] [Questions] [answered]
Hello.
Maybe, stupid question, but still:
how to enable all logs which is available?
I see INFO, CRITICAL and DEBUG messages, but do not see all access logs and error logs,
I added these environment variables:
```
LOG_LEVEL: "debug"
ACCESS_LOG: "-"
ERROR_LOG: "-"

**Comment by tiangolo** [comment, marked as answer]
I'm not sure I understand the question. :thinking: 

But anyway, just so you know...

Now that Uvicorn supports managing workers with `--workers`, including restarting dead ones, there's no need for Gunicorn. That also means that it's much simpler to build a Docker image from scratch now, I updated the docs to explain it.

Because of that, I deprecated this Docker image: https://github.com/tiangolo/uvicorn-gunicorn-docker#-warning-you-probably-dont-need-this-docker-image
